src/hooks/screen/useChatScreenHook.ts

```diff
@@ -293,20 +297,12 @@ function useChatScreenHook(type: 'SIGNED' | 'ANONYMOUS'): UseChatScreenHook {
       }
       if (Platform.OS === 'android') {
         await currentChatSchema?.updateChatSentStatus(localDb, response);
-        initChatData();
+        handleChatData();
         refresh('channelList');
       } else {
-        queue.addPriorityJob({
-          operationLabel: DatabaseOperationLabel.ChatScreen_UpdateChatSentStatus,
-          id: `${selectedChannel?.id}-${randomId}`,
-          priority: QueueJobPriority.HIGH,
-          forceAddToQueue: true,
-          task: async () => {
-            await currentChatSchema?.updateChatSentStatus(localDb, response);
-            initChatData();
-            refresh('channelList');
-          }
-        });
+        await currentChatSchema?.updateChatSentStatus(localDb, response);
+        handleChatData();
+        refresh('channelList');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, the `handleChatData` function is called twice in this code block, once for each branch of the if-else statement. Consider calling this function once after the if-else statement.
```diff
  if (Platform.OS === 'android') {
    await currentChatSchema?.updateChatSentStatus(localDb, response);
-   handleChatData();
    refresh('channelList');
  } else {
    await currentChatSchema?.updateChatSentStatus(localDb, response);
-   handleChatData();
    refresh('channelList');
  }

+ handleChatData();
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->